src/subscriptions/__tests__/subscriptionsReducer-test.js

```diff
@@ -1,46 +1,67 @@
+/* @flow strict-local */
+
 import deepFreeze from 'deep-freeze';
 
+import type { Subscription } from '../../api/modelTypes';
 import { EventTypes } from '../../api/eventTypes';
-import { REALM_INIT, EVENT_SUBSCRIPTION, ACCOUNT_SWITCH, EVENT } from '../../actionConstants';
+import { EVENT_SUBSCRIPTION, ACCOUNT_SWITCH, EVENT } from '../../actionConstants';
 import subscriptionsReducer from '../subscriptionsReducer';
+import * as eg from '../../__tests__/lib/exampleData';
+
+/** Boring properties common to all example Subscription objects. */
+const subscriptionPropertiesBase: $Shape<Subscription> = deepFreeze({
+  color: '#888888',
+  in_home_view: false,
+  pin_to_top: false,
+  audible_notifications: false,
+  desktop_notifications: false,
+  email_address: `${eg.randString()}@example.org`,
+  is_old_stream: false,
+  push_notifications: null,
+  stream_weekly_traffic: 0,
+});
+
+const subscriptionPropertiesFromStream: $Shape<Subscription> = deepFreeze(eg.stream);
+
+export const makeSubscription = (args: $Shape<Subscription> = {}): Subscription =>
```

Comment:
Note: we may instead want to cherry-pick https://github.com/zulip/zulip-mobile/commit/947dd8d26e71d8b48714ede3e24f09f9cae80263 (not merged) from #3946.